### PR TITLE
Update App to 0.21.0 version

### PR DESCRIPTION
**Pull request by ljakimczuk**

### CODE REVIEW

File: helm/flux-app/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
-appVersion: 0.19.0
+appVersion: 0.20.1
```
Comment (rossf7) [inline helm/flux-app/Chart.yaml:2]: I think we need to bump the version via kustomize. See https://github.com/giantswarm/flux-app/pull/50

The upstream YAML is defined here https://github.com/giantswarm/flux-app/blob/2a63d57608e1943ae671196f3e2952d8deceec38/hack/kustomization.yaml#L27

Comment (ljakimczuk) [reply helm/flux-app/Chart.yaml]: Thanks, done!

**Comment by rossf7** [comment]
@ljakimczuk This is looking good but can we also get the metadata sorted in Chart.yaml as per   https://github.com/giantswarm/giantswarm/issues/19461

Screen Shot 2021-11-02 at 5 53 06 PM

Let's leave these for now

 - `No kubeVersion given` - it should be optional
 - `No maintainers given` - I don't think this applies.
 - `App name should not end with -app` this is a bigger change.

File: helm/flux-app/Chart.yaml [hunk 2]
```diff
@@ -1,10 +1,19 @@
...
 engine: gotpl
 home: https://github.com/giantswarm/flux-app
 icon: https://s.giantswarm.io/app-icons/flux/1/dark.svg
+keywords:
+- gitops
+- flux
 name: flux-app
 sources:
 - https://github.com/fluxcd/flux2
+type: application
```
Comment (rossf7) [inline helm/flux-app/Chart.yaml:18]: Nice to include this too. 👍

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+## Update from upstream
```
Comment (rossf7) [inline CONTRIBUTING.md:1]: Good to move this. What about adding a contributing section to the readme and linking here?

**Review by rossf7 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (ljakimczuk) [reply CONTRIBUTING.md:1]: Good idea 👍

**Comment by rossf7** [comment]
@marians FYI for https://github.com/giantswarm/giantswarm/issues/19663 the metadata improvements will be in the next release.

File: helm/flux-app/Chart.yaml (hunk 2 above)
Comment (piontec) [reply helm/flux-app/Chart.yaml:18]: Let's make sure we add our custom metadata as well. Can this chart be deployed multiple times per namespace / cluster?
https://intranet.giantswarm.io/docs/product/architecture-specs-adrs/specs/managed-apps/2020-05-05-app-versions-representation/#supported-chartyaml-file-format

**Review by MarcelMue — APPROVED** [review]

**Review by piontec — APPROVED** [review]
LGTM!

**Pull request merged**